internal/tester.go

```diff
@@ -79,63 +94,120 @@ func (t *Tester) ExecuteTests() error {
 }
 
 func executeSingleTestFile(t *Tester, tf string, timeout time.Duration, resources []config.Resource) error {
-	chainsawCommand := fmt.Sprintf(`"${CHAINSAW}" test --test-dir %s --test-file %s --skip-delete --parallel 1 2>&1`,
-		filepath.Clean(filepath.Join(t.options.Directory, caseDirectory)),
-		filepath.Clean(tf))
+	// Explicitly Set Controller Logger
+	// because of log.SetLogger(...) was never called;
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
-	ctx, cancel := context.WithTimeout(context.Background(), timeout)
-	defer cancel()
+	log.Println("Loading default configuration...")
+	configuration, err := kconfig.DefaultConfiguration()
+	if err != nil {
+		return errors.Wrap(err, "failed to load Chainsaw default configuration")
+	}
+	configuration.Spec.Discovery.TestFile = tf
+	configuration.Spec.Execution.Parallel = ptr.To(1)
+	configuration.Spec.Cleanup.SkipDelete = true
+
+	log.Printf("- Using test file: %s\n", configuration.Spec.Discovery.TestFile)
+	log.Printf("- ApplyTimeout %v\n", configuration.Spec.Timeouts.Apply.Duration)
+	log.Printf("- AssertTimeout %v\n", configuration.Spec.Timeouts.Assert.Duration)
+	log.Printf("- CleanupTimeout %v\n", configuration.Spec.Timeouts.Cleanup.Duration)
+	log.Printf("- DeleteTimeout %v\n", configuration.Spec.Timeouts.Delete.Duration)
+	log.Printf("- ErrorTimeout %v\n", configuration.Spec.Timeouts.Error.Duration)
+	log.Printf("- ExecTimeout %v\n", configuration.Spec.Timeouts.Exec.Duration)
+	log.Printf("- Parallel %d\n", *configuration.Spec.Execution.Parallel)
```

Comment:
We have some timeout override mechanisms in the template and example manifest files. I am curious about, can we have still this capability after the changes? Did we check this?

Reply:
we have 2 layers of configuration - default configuration and test specific overrides - we let chainsaw library handling the overrides like before